use-cases.md

```diff
@@ -69,11 +65,7 @@ The `operation` above refers to the name the server wants to give to the Span. F
 The `span` object above can be `None` if the Tracer did not find relevant headers in the incoming request: presumably because the client did not send them. In this case the server needs to start a brand new trace.
 
 ```python
-    span = tracer.join_trace_from_text(
-        operation_name=operation,
-        context_snapshot=request.headers,
-        trace_attributes=request.headers
-    )
+    span = tracer.get_span_extractor(request.class).extract(request)
```

Comment:
I think what you're saying is that the custom end-user written codec will not be an `f(span, target)` but an `f(span, target, g)`, where `g = tracer.injector(map or binary)`, i.e. it will have to use other (more standard) injector from tracer into a known & shared data model, and then read from that model into the `target` which is unknown to the tracer.

```
func f(span, target) {
    g := tracer.injector(map[string]string)
    m := map()
    g.inject(span, m)
    // now do m => target
}
```